src/Server/Server.fsproj

```diff
@@ -20,10 +20,7 @@
       <Paket>True</Paket>
     </Content>
 
-    <PackageReference Include="FSharp.NET.Sdk" Version="1.0.0-beta-060000" PrivateAssets="All" />
-    <DotNetCliToolReference Include="dotnet-compile-fsc" Version="1.0.0-preview2-020000" />
-
-    <EmbeddedResource Include="**\*.resx" />
+    <PackageReference Include="FSharp.NET.Sdk" Version="1.0.*" PrivateAssets="All" />
```

Comment:
Sorry @sergey-tihon, we cannot change this until there's proper IDE support for FSharp.NET.Sdk 1.0. Please, see #28 and https://github.com/dotnet/netcorecli-fsc/wiki/.NET-Core-SDK-1.0.1#ide-support